### ISSUE TITLE
Remove use of prototype extensions

### ISSUE THREAD

**Issue by runspired** [mharris717/ember-cli-pagination#162]
Ember addons which use prototype extensions break in apps which don't.

This may not be the only place they are in use, but it's one:
https://github.com/mharris717/ember-cli-pagination/blob/master/addon/local/paged-array.js#L19-L23

**Comment by broerse** [comment]
You have a point. Do you know how we can set `EmberENV.EXTEND_PROTOTYPES = false;` in tests?

**Comment by runspired** [comment]
All you need to do is use the addon.

`npm install --save-dev ember-disable-prototype-extensions`.

You can replace computed properties with `Ember.computed(...keys, method)`,

**Comment by broerse** [comment]
Yes I saw that but like to test with and without. Will take a closer look.

**Comment by runspired** [comment]
If you have the addon setup to work without prototype extensions, it will work with them as well.  You don't need to test both scenarios.

**Comment by mharris717** [comment]
the setting is easy in the addon as well

**Comment by mharris717** [comment]
Have a PR almost ready for this

**Comment by broerse** [comment]
Nice!

**Comment by broerse** [comment]
@runspired applied @mharris717 PR to switch to ember-cli 2.7.0 and remove bindings. @runspired can you test this?
